sycl/include/CL/sycl/group.hpp

```diff
@@ -104,35 +104,110 @@ template <int Dimensions = 1> class group {
 
   group() = delete;
 
+  __SYCL_DEPRECATED("use sycl::group::get_group_id() instead")
```

Comment:
could you add a test case to https://github.com/intel/llvm/blob/sycl/sycl/test/warnings/sycl_2020_deprecations.cpp to check deprecation warning for removed API?

Reply:
Done